clients/python-wrapper/lakefs/config.py

```diff
@@ -68,20 +69,27 @@ def __init__(self, verify_ssl: Optional[bool] = None, proxy: Optional[str] = Non
                 self.server = ClientConfig.Server(**data["server"])
                 self.credentials = ClientConfig.Credentials(**data["credentials"])
             found = True
+        except KeyError:	# in case credentials or server is not elaborated
```

Comment:
Seems like this part might be buggy:
1. If server information exists and credentials are not provided - self.server will be populated
2. If credentials infromation exist but server is not provided - nothing will be populated
Seems like inconsistent behavior.
Also there's a variety of errors that can come from parsing the lakectl yaml file, I feel like this is putting a band aid instead of trying to fix the actual problem.
There should be a check to see if the key exists first and then try to create the configuration instead of just ignoring the exception.